docs/use.md

```diff
@@ -0,0 +1,106 @@
+
+```
+github-activity jupyter/notebook -s 6.0.0 -u 6.0.1 -o sample_notebook_activity.md
+```
+
+You can find the [resulting markdown here](sample_notebook_activity).
+
+```{tip}
+For repositories that use multiple branches, it may be necessary to filter PRs by a branch name.  This can be done using the `--branch` parameter in the CLI.   Other git references can be used as well in place of a branch name.
+```
+
+### Split PRs by tags and prefixes
+
+Often you wish to split your PRs into multiple categories so that they are easier
+to scan and parse. You may also _only_ want to keep some PRs (e.g. features, or API
+changes) while excluding others from your changelog.
+
+`github-activity` uses the GitHub tags as well as PR prefixes to automatically
+categorize each PR and display it in a section in your markdown. It roughly
+follows the [keepachangelog taxonomy of changes](https://keepachangelog.com/en/1.0.0/).
+
+Below is a list of the supported PR types, as well as the tags / title prefixes
+that will be used to identify the right category.
+
+```{include} _build/dirhtml/tags_list.txt
+```
+
+```{tip}
+You can choose to *remove* some types of PRs from your changelog by passing the
+`--tags` parameter in the CLI. This is a list of a subset of names taken from the
+left-most column above.
+```
+
+## Use a GitHub API token
+
+`github-activity` uses the GitHub API to pull information about a repository's activity.
+You will quickly hit your API limit unless you use a personal access token.
+There are two ways that you can generate your own access token for use with `github-activity`, each is described below:
+
+### Create a token using the GitHub CLI
+
+You can use [the GitHub command line interface](https://cli.github.com/manual/) to authenticate your account and store an access token in your local environment.
+To do so, download the GitHub CLI, and run the following command:
+
+```bash
+# Authenticate with GitHub via the web interface
+gh auth login --web
+```
+
+This will open a web interface for you to authenticate.
+When finished, it will store an access token locally, which you can print with:
+
+```bash
+# Print an access token if it is stored
+gh auth status -t
+```
+
+This token will automatically be used by `github-activity` if it exists.
+
+### Manually create your own API token
+
+Alternativelly, you can create your own GitHub access token and store it yourself.
+To do so, follow these steps:
+
+- Create your own access token. Go to the [new GitHub access token page](https://github.com/settings/tokens/new)
+  and follow the instructions. Note that while working with a public repository,
+  you don't need to set any scopes on the token you create.
+- When using `github-activity` from the command line, use the `--auth` parameter and pass
+  in your access token. This is easiest if you set it as an **environment variable**,
+  such as `MY_ACCESS_TOKEN`. You can then add it to your call like so:
```

Comment:
From a security perspective, especially on a shared system, it's safer to set the `GITHUB_ACCESS_TOKEN` env-var instead of passing secrets on the command line